docs/devsguide/new_dev_guide.rst

```diff
@@ -1,133 +1,213 @@
+If you’re on Fedora, for example, you can use yum: ::
 
 $ sudo yum install git
 
-If you’re on a Debian-based distribution like Ubuntu, try apt-get:
+If you’re on a Debian-based distribution like Ubuntu, try apt-get: ::
 
 $ sudo apt-get install git
 
-For more options, there are instructions for installing on several different 
+For more options, there are instructions for installing on several different
 Unix flavors on the Git website, at http://git-scm.com/download/linux.
 
-==================================
+--------------------------------
 Connecting Github account to Git
-==================================
-More detailed steps go here 
-‘Github<https://help.github.com/categories/bootcamp/>’
+--------------------------------
+For more detailed steps go `here 
+<https://help.github.com/categories/bootcamp/>`__.
+
+Overview:
+
+1. In your computer's terminal, tell Git your name so your commits will be
+properly labeled. ::
+    $ git config --global user.name "YOUR NAME"
+2. Tell Git the email address that will be associated with your Git commits. The
+email you specify should be the same one found in your email settings on Github. ::
+    $ git config --global user.email "YOUR EMAIL ADDRESS"
+3. To contribute to the project, Fork PyNE’s `Github repository 
+<https://github.com/pyne/pyne>`__.
 
-#. In your computer's terminal, tell Git your name so your commits will be 
-properly labeled. Type everything after the $ here:
+Learn how to keep your email address hidden `here
+<https://help.github.com/articles/keeping-your-email-address-private/>`__.
 
-$ git config --global user.name "YOUR NAME"
+================
+Installing PyNE:
+================
 
-#. Tell Git the email address that will be associated with your Git commits. The 
-email you specify should be the same one found in your email settings on Github. 
-To keep your email address hidden click 'here 
-<https://help.github.com/articles/keeping-your-email-address-private/>'_ 
+There are a couple of methods outlined for downloading PyNE on this site
+under `Installation <https://pyne.io/install/index.html>`__. For
+developers, it is recommended that you install PyNE from the source.
 
-$ git config --global user.email "YOUR EMAIL ADDRESS"
+Make sure that your device has all of the dependencies downloaded. You 
+can check this by typing the name of the program into your command line 
+followed by "-v" (e.g. $ python -v). If you are just starting as a 
+developer, likely, you will not have all of the necessary components.
 
-==================================
-Installing from Source
-==================================
-NOTE: Might have to install pip, home-brew, and other programs to correctly 
-install.
-      Also, installing PyNE isn’t the same as cloning the repository.
+A particularly important detail to pay attention to is the version of
+each dependency. For example, Python is currently supported in conda
```

Comment:
Do you mean "PyNE is currently supported in ...."?

Reply:
Yes, that was my intention. Thank you for the note